screens/Contacts/ContactListScreen.tsx

```diff
@@ -0,0 +1,125 @@
+  {uid: '5', firstName: 'Charles', lastName: 'Dupond', profilePictureUrl: '', description: 'Aaaaaaaaaaaaaaaaaaaa v v v v v v v v v a a a a a a a a a a a a a a a a a a a a a a a aaa a a a a a a a a a a a a a aa a a a a a a a a a a a a a a a a a a a a a a aaa a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a aaa a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a aaa a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a aaa a a a a a a a a a a a a a a a a a a a a a a a a a a a a a a', interests: ['machine learning'], qualifications: ['running', 'bowling']},
+  {uid: '6', firstName: 'Charlotte', lastName: 'Motte', profilePictureUrl: '', description: 'Bbbbbbbbbbbbbbbbbb', interests: ['walking'], qualifications: ['running']},
+  {uid: '7', firstName: 'Charlović', lastName: 'Poutine', profilePictureUrl: '', description: 'Description 7', interests: [], qualifications: ['python', 'java', 'C++']},
+  {uid: '8', firstName: 'Stephanović', lastName: 'Vladimir', profilePictureUrl: '', description: 'I love fish', interests: ['surfing', 'machine learning'], qualifications: ['biology']},
+  {uid: '9', firstName: 'Lulu', lastName: 'Nom', profilePictureUrl: '', description: 'I love bamboo', interests: ['surfing', 'machine learning'], qualifications: ['litterature']},
+  {uid: '10', firstName: 'Lili', lastName: 'De', profilePictureUrl: '', description: '19 + 4 = 22', interests: ['surfing', 'machine learning'], qualifications: ['a', 'b', 'c', 'd', 'e', 'f', 'h']},
+  {uid: '11', firstName: 'Lala', lastName: 'Famille', profilePictureUrl: '', description: 'I\'m doing university for fun', interests: ['a', 'b', 'c', 'd', 'e', 'f', 'h'], qualifications: ['a', 'b', 'c', 'd', 'e', 'f', 'h']},
+  {uid: '12', firstName: 'Abc', lastName: 'Onu', profilePictureUrl: '', description: 'Didn\'t want a description', interests: [], qualifications: ['a', 'b', 'c', 'd', 'e', 'f', 'h']},
+  {uid: '13', firstName: 'Def', lastName: 'Steph', profilePictureUrl: '', description: '-', interests: ['movies'], qualifications: ['history']},
+  {uid: '14', firstName: 'Hij', lastName: 'Non', profilePictureUrl: '', description: 'Mataphisical question', interests: [], qualifications: []},
+]
+
+export const ContactListScreen = () => {
+  const [filteredContacts, setFilteredContacts] = useState(dummyData);
+  const [searchText, setSearchText] = useState('');
+  const [selectedTab, setSelectedTab] = useState('Plain View');
+ 
+  const handleSearch = (text: string) => {
+    setSearchText(text);
+    if(text){
+        const filtered = dummyData.filter((contact) =>
+          `${contact.firstName} ${contact.lastName}`.toLowerCase().includes(text.toLowerCase())
+        )
+        setFilteredContacts(filtered);
+    }else{
+        setFilteredContacts(dummyData);
+    }
+  }
+
+  const RenderOneContact = ({ item } : { item : Contact }) => (
+    <View style={styles.cardContainer}>
+      {item.profilePictureUrl? (
```

Comment:
Here I'm still using an image and an icon for the default "no profile picture". I believe this is the way to go.

Reply:
Agree, Ionicons is the way to go